src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REM_BOOK = 'REM_BOOK';
```

Comment:
- [ ] Kindly match the duck pattern, it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`. Kindly update it.